templates/docs/patterns/grid.md

```diff
@@ -12,9 +12,9 @@ Vanilla has a responsive grid with the following columns and gutters:
 
 | Screen size (px)                       | Columns | Grid gap (gutters) | Outer margins |
 | -------------------------------------- | ------- | ------------------ | ------------- |
-| 0 - \$breakpoint-small                 | 4       | 1.5rem             | 1.0rem        |
+| 0 - $breakpoint-small                  | 4       | 1.5rem             | 1.0rem        |
 | $breakpoint-small - $breakpoint-medium | 6       | 2.0rem             | 1.5rem        |
-| above \$breakpoint-medium              | 12      | 2.0rem             | 1.5rem        |
+| above $breakpoint-medium               | 12      | 2.0rem             | 1.5rem        |
```

Comment:
I think this would be better as `less than $breakpoint-small` and `greater than $breakpoint-medium`

Reply:
Yeah, makes sense. I've changed it to your suggestion